Locations

Jackie-J-J opened the pull request:
1. Created 'locations' app. 
   Currently, the ‘locations’ app allows you to view your current location when you visit http://localhost:8000/locations/show-location/
   The 'locations' app uses HTML5 Geolocation API retrieves the user’s current latitude and longitude and uses Leaflet.js to display a map.
2. Added localhost to ALLOWED_HOST in routepals/settings.py

jeffwong97 commented:
Hey @Jackie-J-J thanks for figuring this out! I looked through the code and just have a couple comments:

- Can you add the installation requirements to requirements.txt and add some instructions to the readme if possible so we can have it working on all our machines
- Can you move your html template into the templates folder that is separate from the apps so all templates are organized in one place

Jackie-J-J commented:
> Hey @Jackie-J-J thanks for figuring this out! I looked through the code and just have a couple comments:
> 
> * Can you add the installation requirements to requirements.txt and add some instructions to the readme if possible so we can have it working on all our machines
> * Can you move your html template into the templates folder that is separate from the apps so all templates are organized in one place

Hey @jeffwong97, thank you very much for your comments!

	1.	Shashank and I have added instructions to the README.md on installing PostgreSQL, extensions, and setting up the database and user, etc. Please feel free to ask if you have any questions.
	2.	I’ve moved show_location.html to the main templates folder.
	3.	I’m not sure how to merge the README.md into the develop branch, so I may need to ask you about that later.
